microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md

```diff
@@ -1,133 +1,211 @@
 ---
-description: How to use Elements for Microsoft Edge (Chromium) from Visual Studio Code
-title: Elements for Microsoft Edge (Chromium) from Visual Studio Code
+description: How to use Microsoft Edge Developer Tools for Visual Studio Code extension
+title: Microsoft Edge Developer Tools for Visual Studio Code extension 
 author: MSEdgeTeam
 ms.author: msedgedevrel
-ms.date: 01/07/2021
+ms.date: 07/27/2021
 ms.topic: article
 ms.prod: microsoft-edge
-keywords: microsoft edge, web development, f12 tools, devtools, vs code, visual studio code, elements
+keywords: microsoft edge, web development, f12 tools, devtools, vs code, visual studio code, microsoft edge developer tools, microsoft edge developer tools extension
 ---
 
-# Microsoft Edge DevTools for Visual Studio Code extension  
+# Microsoft Edge Developer Tools for Visual Studio Code extension  
 
-Use <!--the [Microsoft Edge DevTools for Visual Studio Code][VisualstudioMarketplaceElementsMicrosoftEdgeChromium] -->this extension to access in Microsoft Edge DevTools inside [Microsoft Visual Studio Code][VisualstudioCode].  You have access to the **Elements** and **Network** tools in Microsoft Edge DevTools.  You may either launch or attach to an instance of Microsoft Edge.  After you connect you may display the runtime HTML structure, alter the layout, fix styling issues, and inspect network traffic.  
+The Microsoft Edge Developer Tools for Visual Studio Code extension enables you to use the browser’s Elements and Network tool from within the editor. 
+Without leaving Visual Studio Code, use Microsoft Edge Developer Tools (DevTools) to connect to an instance of Microsoft Edge with the following abilities.
+* See the runtime HTML structure.
+* Change the layout.
+* Change styles (CSS).
+* Read console messages.
+* View network requests. 
 
-## Elements tool  
+> [!NOTE]
+> This extension only works with Microsoft Edge (versions 80.0.361.48 and later).
 
-By default, the extension launches a browser instance in a unique window and gives you the **Elements** functionality of Microsoft Edge DevTools.  
+There are two ways to open Microsoft Edge Developer Tools.
+* From the activity bar in Visual Studio Code.
+* From the JavaScript Debugger in Visual Studio Code.
 
-:::image type="complex" source="./media/edge-devtools-for-vscode-windowed.png" alt-text="Microsoft Edge DevTools for Visual Studio Code running with a full browser window" lightbox="./media/edge-devtools-for-vscode-windowed.png":::
-   Microsoft Edge DevTools for Visual Studio Code running with a full browser window  
-:::image-end:::  
+This article describes the Microsoft Edge Developer Tools for Visual Studio Code extension. You can [download the extension][https://marketplace.visualstudio.com/items?itemName=ms-edgedevtools.vscode-edge-devtools] and [check the source code][https://github.com/microsoft/vscode-edge-devtools].
```

Comment:
Can we fix the links? [pagename][linkname] and then add the link details in the section at the bottom of the page.